examples/sdk-examples/src/examples/execution/ExecuteWithCustomVisualizationExample.tsx

```diff
@@ -0,0 +1,70 @@
+// (C) 2007-2019 GoodData Corporation
+import React from "react";
+import { Execute, LoadingComponent, ErrorComponent } from "@gooddata/sdk-ui";
+import { ResponsiveContainer, BarChart, CartesianGrid, XAxis, YAxis, Legend, Bar } from "recharts";
+
+import * as LdmExt from "../../ldm/ext";
+
+const seriesBy = [
+    LdmExt.FranchiseFeesAdRoyalty,
+    LdmExt.FranchiseFeesInitialFranchiseFee,
+    LdmExt.FranchiseFeesOngoingRoyalty,
+];
+
+const slicesBy = [LdmExt.monthDate];
+
+const colors = ["rgb(20,178,226)", "rgb(0,193,141)", "rgb(229,77,66)"];
+
+export const ExecuteWithCustomVisualizationExample: React.FC = () => {
+    return (
+        <Execute seriesBy={seriesBy} slicesBy={slicesBy}>
+            {({ isLoading, error, result }) => {
+                if (isLoading) {
+                    return <LoadingComponent />;
+                }
+
+                if (error) {
+                    return (
+                        <ErrorComponent
+                            message="There was an error getting your execution"
+                            description={JSON.stringify(error, null, 2)}
+                        />
+                    );
+                }
+
+                const series = result?.data().series().toArray();
+                const slices = result?.data().slices().toArray();
+
+                const bars = series?.map((value, index) => {
+                    return (
+                        <Bar
+                            key={value.id}
+                            dataKey={value.id}
+                            name={value.measureTitle()}
+                            fill={colors[index]}
+                        />
+                    );
+                });
+
+                const data = slices?.map((slice) => ({
+                    label: slice.sliceTitles()[0],
+                    ...slice.dataPoints().map((p) => p.rawValue),
```

Comment:
is it possible to use formatted data here? or the chart only supports numbers?

Reply:
I'm not sure, let me check

Reply:
Nope, the library only supports numbers.